### DISCUSSION TITLE
Print Review, too thin areas

### DISCUSSION THREAD

**Discussion by bosd** [joshreve/dactyl-keyboard#92] [General]
I received feedback from the jlcpb printing service, which I'm trying out.
Very glad that a print service reviews the design. (As my previous print was wasted)
Some of the items, I could fix, but not sure, if I am able to alter the thickness of the bottom plate.

Well leaving this here for you all to learn.
![bottom plate](https://user-images.githubusercontent.com/11499387/173293199-cdc7c6d3-5dae-43a6-bf24-dcbf223f6f30.png)
![bottomplate2](https://user-images.githubusercontent.com/11499387/173293219-92f70819-eb1f-4f4f-adb7-134d309f6daa.png)
![detail](https://user-images.githubusercontent.com/11499387/173293245-937bbe26-c379-4645-a6dc-467afde019ef.png)
![elite c holder](https://user-images.githubusercontent.com/11499387/173293284-505c6377-43e2-4519-baa2-37721bf2d970.png)
![left](https://user-images.githubusercontent.com/11499387/173293309-c003498b-86bf-471d-9aa1-7fd164ad2e11.png)

![clip](https://user-images.githubusercontent.com/11499387/173293232-ee97983d-9053-4b51-8f39-75141947bb6f.png)

**Comment by bosd** [comment]
The previous print the  treatstock seller made for me was had damaged areas near the x wall screw on the bottom plate.
These are ineed to close to the wall.
So started modding...

This is wall_bae_thick *2
![image](https://user-images.githubusercontent.com/11499387/173450678-351089ae-90c3-4193-8351-dfd07d8ef2aa.png)
screw inserts are dangling in air.
x1,5
![image](https://user-images.githubusercontent.com/11499387/173452589-9ab40dba-2df9-433b-b833-fb1e2da10a9c.png)
eventually went for:

![image](https://user-images.githubusercontent.com/11499387/173460949-ebd0633d-1f44-4755-b879-2a199f5a8dbc.png)

Was expecting to have the screw inserts relative from the wall. e.g. so that the distance of the insert in the wall is always fixed.
Somehow it works differently. as you can tell the screw inserts are not on the same line.

Still feels a bit like a hack, but opened a pr, to move the x screw inserts just a bit inward. So the result is like in the last screenshot.
It does not look like it. But the 6.5mm must be enough to have a solid screwinsert attacchment.

**Reply by joshreve** [reply]
The screws have always been a hack on every version of this I've seen.  The issue is that the way it is constructed, it doesn't "understand" the z=0 plane created during a later cut to flatten the bottom.  There are no direct reference points to items on that plane so the screws are best estimate based on the default structure.  I believe I added a place to shift them for this reason.  When I had more time to work on this, I started a version with more "self awareness", but it was not a trivial task and I had to move on as it would be a dramatic restructuring.  

**You are right to tweak the screws in place.** 

The hotswap holders have also always been finicky.  They want PCB thickness, which is a bit thin for printing.  I ended up wiring howswaps, but pushing them on.  Not as easy to change, but I found I just didn't like the fit of the thin holders.  I added them because other did and it wasn't too hard to put them in for people to use.

I actually print mine at a wierd angle to minimize support.  I can't remember exactly, but I kept messing with the angle in the slicers until I got most of it self supported at 40 degrees or steeper, and used tree supports to minimize the extra support material.  Printing these to get the best quality is definitely an art given all the contours.

**Comment by bosd** [comment]
Fix for the bottom plate isin #93
